CHANGELOG.md

```diff
@@ -20,6 +21,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - [#667](https://github.com/nf-core/mag/pull/667) - Fix pipeline crashing if only CONCOCT selected during binning (reported and fixed by @jfy133)
 - [#670](https://github.com/nf-core/mag/pull/670) - Re-add missing GTDBTk parameters into GTDBTk module (reported by harper357, fixed by @jfy133)
 - [#672](https://github.com/nf-core/mag/pull/673) - Fix GTDB-Tk per-sample TSV files not being published in output directory (reported by @jhayer, fix by @jfy133)
+- Make longread preprocessing a subworkflow
+- Add porechop and filtlong logs to multiqc
+- Change local filtlong module to the official nf-core/filtlong module
```

Comment:
```suggestion
- [#674](https://github.com/nf-core/mag/pull/674/files) -  Make longread preprocessing a subworkflow  (added by @muabnezor)
- [#674](https://github.com/nf-core/mag/pull/674/files) -  Add porechop and filtlong logs to multiqc  (added by @muabnezor)
- [#674](https://github.com/nf-core/mag/pull/674/files) -  Change local filtlong module to the official nf-core/filtlong module (added by @muabnezor)
```

Reply:
I would also add an entry saying switched to `porechop_abi` as default long-read adapter removal tool (See my comment below).

it is faster so should be better. We keep the old one as fallback if people don't trust the new one.